excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ListFieldTypeParser.java

```diff
@@ -81,6 +84,14 @@ public ListFieldTypeParser(ParseContext parseContext) {
         parseContext.addToAllTypes(fixedListType);
         result.addNew(fixedListType);
 
+        // Add as DynamicList with no list items as it will be dynamically added by the services
+        final FieldTypeEntity dynamicListType = new FieldTypeEntity();
```

Comment:
the ticket says:
```If a list has been been defined using 'FieldTypeParameter' when it is 'DynamicList' it configured list should be ignored
In this case do we really need this block of code here then?

Reply:
Removed check for fieldtypeparameter changes for DynamicList